cluster/images/etcd/Makefile

```diff
@@ -43,8 +43,8 @@ ARCH?=amd64
 
 # Operating systems supported: linux, windows
 OS ?= linux
-# OS Version for the Windows images: 1809, ltsc2022
-OSVERSION ?= 1809
+# OS Version for the Windows images: 1809 ltsc2019, ltsc2022
```

Comment:
If we're not replacing 1809 with ltsc2019, we should note here that 1809 is deprecated as well, just as it is above on the pause image. Also, you're missing a ``,`` after 1809.

Reply:
Updated.